stf/feature_engineering/feature_free_days.py

```diff
@@ -6,11 +6,13 @@
 
 import numpy as np
 import pandas as pd
+from datetime import date
+import holidays
 
 HOLIDAY_CSV_PATH = os.path.dirname(__file__) + "/dutch_holidays_2020-2022.csv"
 
 
-def create_holiday_functions():
+def create_holiday_functions(country="FR", years=None):
```

Comment:
Probably temporarily, but I don't think it makes sense to use one country as default.